docs/docs/api-reference/starlark-reference/docstring-syntax.md

```diff
@@ -0,0 +1,140 @@
+        service_name (string): The name to give the Postgres service
+        user (string): The user to create the Postgres database with
+        password (string): The password to give to the created user
+        database (string): The name of the database to create
+        config_file_artifact_name (string): The name of a files artifact that contains a Postgres config file in it
+            If not empty, this will be used to configure the Postgres server
+        seed_file_artifact_name (string): The name of a files artifact containing seed data
+            If not empty, the Postgres server will be populated with the data upon start
+        extra_configs (list[string]): Each argument gets passed as a '-c' argument to the Postgres server
+        persistent (bool): Whether the data should be persisted. Defaults to True; Note that this isn't supported on multi node k8s cluster as of 2023-10-16
+        launch_adminer (bool): Whether to launch adminer which launches a website to inspect postgres database entries. Defaults to False.
+    Returns:
+        An object containing useful information about the Postgres database running inside the enclave:
+        ```
+        {
+            "database": "postgres",
+            "password": "MyPassword1!",
+            "port": {
+                "application_protocol": "postgresql",
+                "number": 5432,
+                "transport_protocol": "TCP",
+                "wait": "2m0s"
+            },
+            "service": {
+                "hostname": "postgres",
+                "ip_address": "172.16.0.4",
+                "name": "postgres",
+                "ports": {
+                    "postgresql": {
+                        "application_protocol": "postgresql",
+                        "number": 5432,
+                        "transport_protocol": "TCP",
+                        "wait": "2m0s"
+                    }
+                }
+            },
+            "url": "postgresql://postgres:MyPassword1!@postgres/postgres",
+            "user": "postgres"
+        }
+        ```
+    """
+```
+
+The syntax for this docstring is [the Google function docstring syntax](https://google.github.io/styleguide/pyguide.html#383-functions-and-methods), with optional PEP-484 type annotations in parentheses:
+
+```python
+def run(plan, required_arg, optional_arg=True, untyped_arg=None):
+    """
+    This is the `run` function description. It can even spill
+    over into multiple lines.
+
+    Args:
+        required_arg (string): This is the first argument description. It is required, and is of string type.
+            Note that argument descriptions can spill into the second line, so long as they're indented.
+        optional_arg (bool): This is the second argument description. It is optional, and it is a boolean-type argument.
+        untyped_arg: This argument doesn't have a PEP-484 type annotation.
+    """"
+```
+
+Several things to note about this docstring syntax:
```

Comment:
@mieubrisse this sounds a little cumbersome, maybe:
"Regarding the docstring syntax, note that:"